### PR TITLE
Changes/additions to Bandjalangic internal clf

### PR DESCRIPTION
**Pull request by d97hah**

### CODE REVIEW

File: languoids/tree/pama1250/sout3135/news1235/band1357/band1339/coas1320/twee1234/minj1242/md.ini [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # -*- coding: utf-8 -*-
 [core]
-name = Minjungbal
+name = Minjangbal
 level = dialect
-iso639-3 = xjb
 hid = xjb
```
Comment (xflr6) [inline languoids/tree/pama1250/sout3135/news1235/band1357/band1339/coas1320/twee1234/minj1242/md.ini]: keeping this duplicates  hid `xjb`

File: languoids/tree/pama1250/sout3135/news1235/band1357/band1339/inla1268/cond1242/gida1240/md.ini [hunk 2]
```diff
@@ -5,7 +5,6 @@ level = dialect
 macroareas = 
 	Australia
 countries = 
-iso639-3 = gih
 hid = gih
```
Comment (xflr6) [inline languoids/tree/pama1250/sout3135/news1235/band1357/band1339/inla1268/cond1242/gida1240/md.ini]: keeping this duplicates hid `gih`

File: languoids/tree/pama1250/sout3135/news1235/band1357/band1339/coas1320/twee1234/minj1242/md.ini (hunk 1 above)
Comment (xflr6) [reply languoids/tree/pama1250/sout3135/news1235/band1357/band1339/coas1320/twee1234/minj1242/md.ini]: should the `hh_ethnologue_comment` section below with `isohid = xjb` stay here?

File: languoids/tree/pama1250/sout3135/news1235/band1357/band1339/inla1268/cond1242/gida1240/md.ini (hunk 2 above)
Comment (xflr6) [reply languoids/tree/pama1250/sout3135/news1235/band1357/band1339/inla1268/cond1242/gida1240/md.ini]: should the hh_ethnologue_comment section below with `isohid = gih` stay here?

**Comment by d97hah** [comment]
The hh_ethnologue_comment in both xjb minj1242 and gih gida1240 should go away (I missed that, thanks). I get lost in the bandjalangic vs bandjalangic-rebase branches (I want to do git checkout bandjalangic-rebase to fix thereon but git says there's no such pathspec), maybe easier if you strip them and merge with master?

Thanks also for removing remaining hid:s in erstwhile dialects, glottolog lff2tree doesn't do that itself it seems.

**Comment by xflr6** [comment]
Thanks, updated `bandjalangic-rebase`: not completely sure if you meant to drop or maybe move the `hh_ethnologue_comment` section -- did the former for now. Just to make sure: maybe have another look at the `identifier` and `endangerment` sections below the changes here (i.e. should they stay or also be dropped or moved/adapted?): https://github.com/clld/glottolog/commit/109221bce327f08c6e29eb1b7c4ffade2cb759e8

I think you might need to `git fetch` before you can `git checkout` an unseen remote branch.

Will add a check for `hid` uniqueness into `glottolog check`. If we want to move `hid`s together with `iso639-3` most of the time, we can maybe already build that into `lff2tree`.

**Comment by xflr6** [comment]
Allright, force-pushed the rebase branch to `bandjalangic` and merged to `master`.

**Comment by xflr6** [comment]
Check for duplicate `hid`s added: https://github.com/clld/glottolog/commit/1860a3fd2a793fe47db662d4d77727e50005a8ab

**Pull request merged**